cmd/main.go

```diff
@@ -10,15 +10,14 @@ import (
 
 	"github.com/spf13/cobra"
 	"k8s.io/cli-runtime/pkg/genericclioptions"
+	"k8s.io/component-base/cli"
 	"k8s.io/kubectl/pkg/cmd/util"
-	"k8s.io/kubectl/pkg/util/logs"
```

Comment:
This package is gone now, see https://github.com/kubernetes/kubernetes/commit/21d1bcd6b8498370832fa76f680d3de56837bc83